src/jit.rs

```diff
@@ -706,9 +723,18 @@ impl<'a> JitMemory<'a> {
             }
         }
 
-        // Initialize registers
-        emit_mov(self, ARGUMENT_REGISTERS[2], R10); // JitProgramArgument
+        // Save JitProgramArgument
+        emit_mov(self, ARGUMENT_REGISTERS[2], R10);
+
+        // Initialize and save BPF stack pointer
         emit_load_imm(self, REGISTER_MAP[STACK_REG], MM_STACK_START as i64 + CALL_FRAME_SIZE as i64);
+        emit_push(self, REGISTER_MAP[STACK_REG]);
+
+        // Padding on the stack to reach 16 byte alignment
```

Comment:
What is being padded and why?

Reply:
RSP must stay 16 byte aligned, thus we push R11 containing the value zero here.
However, this slot on the stack will be used by the instruction meter in the next PR,
so this workaround is very short lived anyway.